docs/4.2.yaml

```diff
@@ -21,6 +21,7 @@ markdown_extensions:
   - footnotes
   - toc:
       marker: '[TOC]'
+      toc_depth: 2
```

Comment:
Love this, didn't realise we could do it.

Reply:
^ Yeah, it' a new-ish feature... I've been a bit slow to adopt the new features after I updated the MkDocs image.